io.github.efogdev.mpris-timer.yml

```
@@ -0,0 +1,38 @@
+id: io.github.efogdev.mpris-timer
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: mpris-timer
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio 
+  - --share=ipc
```

```suggestion
  - --share=ipc
  - --device=dri
```

dri is needed as these has a GUI

---

done